winpkg/ddev_lib_test.nsi

```diff
@@ -0,0 +1,84 @@
+Name "DDEV Lib Test"

```

Comment:
This seems like a nice thing. 

I do suspect that this will be bundled into the release build by accident. 

Is there a way that we can extend or use this as part of "make test"?

Reply:
I created this to test the two libs and thought I will include it for later manual tests after changes in the libs. For the normal build process this it not needed ofc.

Reply:
It's currently not possible to use this test automated, you have to compile the installer and run it. Then verify the output of the installer...

Reply:
We could do that on the Windows test runners. It has potential for the future.